File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/ArbeidIFlereLand.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class ArbeidIFlereLand {

    private Bosted bosted;

    private Yrkesaktivitet yrkesaktivitet;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/ArbeidIFlereLand.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class ArbeidIFlereLand(
    var bosted: Bosted? = null,
    var yrkesaktivitet: Yrkesaktivitet? = null
)
